src/tree/OpenUrlTreeItem.ts

```diff
@@ -3,22 +3,26 @@
  *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzExtParentTreeItem, GenericTreeItem } from "vscode-azureextensionui";
+import * as vscode from 'vscode';
 import { openExternal } from "../utils/openExternal";
-import { getThemedIconPath } from "./IconPath";
 
-export class OpenUrlTreeItem extends GenericTreeItem {
+export interface OpenUrl {
+    openUrl(): Promise<void>
+}
+
+export class OpenUrlTreeItem extends vscode.TreeItem implements OpenUrl {
```

Comment:
What's wrong with `GenericTreeItem`?

Reply:
I tried to use the existing one but that require to pass a parent tree item and have dependency on AzExtTreeItem. Hence a new one that has no dependency on AzExtTreeItem and also allows using icons from vscode.

Reply:
You can probably make VSCode icons work, if not I'd submit a PR to vscode-azuretools. In any case, we should use the established pattern since it does everything we need it to without all these extra classes and files.

Reply:
Why unnecessarily use AzExtTreeItem require implementing AzExtParentTreeItem which require implementing loadMoreChildrenImpl and registering command *.loadMore. These are simple links and commands and doesn't require the functionality provided by the AzExtTreeItem which adds complexity.

Reply:
I will try to use AzExtTreeItem and see how it goes.